sfge/src/main/java/com/salesforce/graph/ops/ApexValueUtil.java

```diff
@@ -33,15 +34,15 @@
 
 public final class ApexValueUtil {
     private static final Logger LOGGER = LogManager.getLogger(ApexValueUtil.class);
+    private static final String UNKNOWN = "UNKNOWN";
 
     /** Currently concatenates string values. TODO: Expand to other operations on Integers etc. */
     public static Optional<ApexValue<?>> applyBinaryExpression(
```

Comment:
Change in order and minor code refactor to perform null access checks on binary vertices.